public/js/app/components/RoundApplicationInfo.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Button } from 'reactstrap'
+import i18n from '../../../../i18n'
+import { INFORM_IF_IMPORTANT_INFO_IS_MISSING } from '../util/constants'
+
+const LABEL_MISSING_INFO = { en: INFORM_IF_IMPORTANT_INFO_IS_MISSING[0], sv: INFORM_IF_IMPORTANT_INFO_IS_MISSING[1] }
+
+const RoundApplicationInfo = ({
+  roundHeader,
+  selectedRoundHeader,
+  userLanguage,
+  round,
+  showRoundData,
+  courseHasRound,
+  fundingType,
+}) => {
+  const { courseRoundInformation: translate, courseLabels: labels } = i18n.messages[userLanguage]
```

Comment:
I see that the original code also has renamed those variables, but I'm not sure it is adding clarity. What do you think?

Reply:
I don't think it adds clarity either but didn't want to remove it without reason, I'll take it away now since we both agree!